src/operator/contrib/bilinear_resize.cc

```diff
@@ -69,23 +70,25 @@ void SpatialUpSamplingBilinearUpdateOutput(mshadow::Stream<cpu> *s,
     }
     return;
   }
+  const DType rheight = area_pixel_compute_scale<DType>(
```

Comment:
We may use float rather than DType, as DType is the data type of inputs and outputs. However, the type of the position and width/height should be always float.
For example, we need resize a feature map whose DType is int32, but the type of width/height and h1r should be float.

Reply:
I'll investigate it. Can we use AccType with this CPU implementation?